Formula/pipx.rb

```diff
@@ -29,13 +29,6 @@ class Pipx < Formula
     sha256 "410e932b050f5eed773c4cda94de75971c89cdb3155a72a0831139a79e5ecb5b"
   end
 
-  resource "distro" do
-    on_linux do
-      url "https://files.pythonhosted.org/packages/a5/26/256fa167fe1bf8b97130b4609464be20331af8a3af190fb636a8a7efd7a2/distro-1.6.0.tar.gz"
-      sha256 "83f5e5a09f9c5f68f60173de572930effbcc0287bb84fdc4426cb4168c088424"
-    end
-  end
-
```

Comment:
removed since this [isn't listed as a requirement](https://github.com/pypa/pipx/blob/3d3f23393aaef7823e032fb359dcf19925d636a2/setup.cfg#L34-L38)